SR2020/2019-11-25-kit-shipment-update.md

```diff
@@ -0,0 +1,12 @@
+---
+to: Those who are awaiting
+subject: Kit Shipment update
+---
+
+Hello!
+
+Unfortunately, We are currently facing delays getting your kit shipped out to you.
```

Comment:
I think it could be useful to discuss why this is. Some of them may be able to either help, or suggest other ways in which we could get the kit to them. Several teams have already asked if they can collect it, for example.

I also don't think there's any reason to withhold the information. In line with our value of being "Open by default", suggest we explain why the delay has happened (that the have been complications in shipping the batteries).